src/contents/en-gb/core_technology.yaml

```diff
@@ -34,6 +34,12 @@ components:
         name: eventual consistency
         href: https://en.wikipedia.org/wiki/Eventual_consistency
       chunk3: "."
+    para3:
+      chunk1: In order to come to this eventual consistency the routing layer uses the consensus mechanism
+      link1:
+        name: PARSEC
+        href: '/parsec'

```

Comment:
This opens the /parsec page in a new browser tab - just confirming whether this is intentional?

Reply:
wasn't just copied and pasted the link line from above but it doesn't make sense to open a new tab. Changed.